experiments/ClimaCore/sea_breeze/run.jl

```diff
@@ -1,18 +1,43 @@
-const FT = Float64
+# # Coupled Sea Breeze Driver
+
+#=
+In this tutorial we demonstrate the coupling of three component models
+(atmosphere, ocean, and land) to drive the sea breeze. The primary parts
+of the ClimaCoupler interface are used and discussed.
+=#
+
 import SciMLBase: step!
 using OrdinaryDiffEq: ODEProblem, solve, SSPRK33, savevalues!
 import ClimaCore.Utilities: PlusHalf
 using DiffEqCallbacks
 
+# enable broadcasting with mismatched spaces
+import ClimaCore: Fields, Operators
+Fields.allow_mismatched_diagonalized_spaces() = true
+Operators.allow_mismatched_fd_spaces() = true
+
 push!(LOAD_PATH, joinpath(@__DIR__, "..", "..", ".."))
 using ClimaCoupler
 
+const FT = Float64
+
+#=
+## Component Models
+Coupled simulations can re-use tendency methods developed for standalone simulations.
+This is achieved by multiple dispatch, where methods that deal with boundaries
```

Comment:
It wasn't clear to me at first that this was referring to `bc_divF2C_bottom!`. It's tough because the "stand-alone" bc in this case is also defined within the coupler. Could we perhaps refer to `bc_divF2C_bottom!` here, or move this to `atmos_rhs.jl`?

Reply:
Yes, perhaps just pointing to a more detailed description in the atmos_rhs.jl docs. I'm not sure how to set up hyperlinks when using Literate. You might need to make Documenter.jl a dependency for the experiment files.